flair/embeddings.py

```diff
@@ -2143,7 +2143,7 @@ def _add_embeddings_internal(self, sentences: List[Sentence]) -> List[Sentence]:
             for token in sentence.tokens:
 
                 # update embedding
-                local_embedding = token._embeddings[self.context_embeddings.name].cpu()
+                local_embedding = token._embeddings[self.context_embeddings.name]
```

Comment:
This fixes the issue, but this would mean that the entire memory would be kept in GPU-memory in a GPU-setup. This would be too much for most GPU setups / datasets.